md_api/src/main/java/com/wipro/ats/bdre/md/api/util/AddJson.java

```diff
@@ -0,0 +1,40 @@
+package com.wipro.ats.bdre.md.api.util;
+
+import com.wipro.ats.bdre.md.rest.beans.ProcessExport;
+import org.codehaus.jackson.map.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Created by cloudera on 3/9/16.
+ */
+public class AddJson {
+
+    public  String addJsonToProcessId(String processId,ProcessExport processExport)
+    {
+
+
+        String homeDir = System.getProperty("user.home");
+
+        String source_dir=homeDir+"/bdre-wfd/"+processId;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00117)